packages/datadog-plugin-graphql/src/index.js

```diff
@@ -63,13 +63,12 @@ function createWrapParse (tracer, config) {
 
         addDocumentTags(span, document)
 
-        finish(span)
-
         return document
       } catch (e) {
         setError(span, e)
-        finish(span)
         throw e
+      } finally {
+        finish(span)
```

Comment:
Isn't this functionally equivalent to the previous code? Just asking to make sure I'm not missing anything, otherwise the change LGTM.

Reply:
It also ensures that `finish()` is called on this early return:

https://github.com/DataDog/dd-trace-js/pull/960/files#diff-50e1c1281cc60a4a3a4dbe04354f9db1L58

Reply:
Ahhh yes sorry I guess I basically forgot about the original issue 😄